VirtoCommerce.Storefront.Model/Stores/Store.cs

```diff
@@ -132,6 +130,14 @@ public bool TaxCalculationEnabled
             }
         }
 
+        public bool AnonymousUsersAllowed
+        {
+            get
+            {
+                return Settings.GetSettingValue("Stores.AllowAnonymousUsers", true);
```

Comment:
Move to StoreConverter.
The same for QuotesEnabled, SubscriptionEnabled, TaxCalculationEnabled.

Reply:
fixed